cleanlab/count.py

```diff
@@ -102,17 +101,17 @@ def calibrate_confident_joint(confident_joint, s, multi_label=False):
     ----------
 
     confident_joint : np.array (shape (K, K))
-        A K,K integer matrix of count(s=k, y=k). Estimates a confident subset of
-        the joint disribution of the noisy and true labels P_{s,y}.
+        A K,K integer matrix of count(labels=k, y=k). Estimates a confident subset of
+        the joint disribution of the noisy and true labels P_{labels,y}.
```

Comment:
Are we sticking with notation `labels` for observed (noisy) labels and `y` for true labels?

Reply:
I changed `s` to `labels` for the most part everywhere. wAre you proposing keeping the notation as `s` in the context of mathematical formula?

Reply:
I'd rather consider changing `y`, since in other ML packages, `y` would refer to the given labels as well. 

Here are various options off top of my head: 
1. keep `labels` as is (or change to `given_labels`, `y` -> `true_labels`)
2. keep `labels` as is (or change to `labels_given`, `y` -> `labels_true`)
3. ``labels` -> `y_given`, `y` -> `y_true` (for the mathematical helper functions only, keep `labels` as arg-name for eg. `num_label_errors()`).
4. `labels` -> `y_observed`, `y` -> `y_actual` (for the mathematical helper functions only, keep `labels` as arg-name for eg. `num_label_errors()`).

Reply:
I didn't use `y` because `y` implies `true labels` in most packages and throughout some of the literature.

Reply:
changing to `p(label=k)` instead of `p(labels=k)` and `p(true_label=k)` instead of `p(true_labels=k)` in mathematical definitions like these. in functions i kept as `labels` and in the very few functions with `y` i changed to `true_labels` (plural).

Reply:
I think this is done. Worth double checking.